### PR TITLE
fix logic so isort and black run when you don't supply "only" args

### PR DESCRIPTION
**Pull request by LyleScott**
it looks like `pyfmt` still needed to be ran against the PR and, since i ran it, this ended up changing some formatting...hence all the little changes. I've noted the actual logic change.

### CODE REVIEW

File: pyfmt/__init__.py [hunk 1]
```diff
@@ -111,14 +118,14 @@ def pyfmt(
 
     # Executing isort import formatter
     isort_exitcode = 0
-    if isort_only:
+    if not black_only:
```
Comment (LyleScott) [inline pyfmt/__init__.py]: these 2 switcharoos is the actual logic change

File: pyfmt/__init__.py [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import os
 import shlex
-from shutil import get_terminal_size
 import subprocess
 import sys
-import time
```
Comment (LyleScott) [inline pyfmt/__init__.py]: unused import

File: pyfmt/__init__.py (hunk 1 above)
Comment (ismet55555) [reply pyfmt/__init__.py]: one thing about this method vs the one I had.... this is good, but if in the future there is another formatter added, you can just add it to the list of `all_format` ... but it's whatever.

Comment (LyleScott) [reply pyfmt/__init__.py]: sure, whatever works here. we don't have another formatter _today_, so maybe roll with less code imho

**Pull request closed without merging**